docs/en/01_Working_with_projects/11_Infrastructural_considerations.md

```diff
@@ -33,7 +33,7 @@ Consider using [caching](how_tos/caching) to speed up request execution.
 
 ## PHP configuration
 
-CWP environments are running PHP 5.6 (see Debian "Jessie" [packages](https://packages.debian.org/jessie/)).
+CWP environments run PHP 5.6 by default (see Debian "Jessie" [packages](https://packages.debian.org/jessie/)). They can be upgraded to run PHP 7.1 via a request to the helpdesk, SilverStripe sites running Framework/CMS version 3.6 and up support PHP 7.
```

Comment:
I had to read the last sentence a few times, do you mind changing it to 
> ... helpdesk. SilverStripe sites running... 

for clarity, please.